synapse/storage/_base.py

```diff
@@ -117,7 +117,7 @@ def _do_execute(self, func, sql, *args):
                 sql, *args
             )
         except Exception as e:
-            logger.debug("[SQL FAIL] {%s} %s", self.name, e)
+            logger.error("[SQL FAIL] {%s} %s", self.name, e)
```

Comment:
Please don't, as this can get really quite spammy iirc

Reply:
From the ticket from @richvdh :

>  if update_client_ips is failing, it needs to log at ERROR or above.

Do we not want to know when SQL fails?

Reply:
Well we raise and handle it later. There are a bunch of failure modes that are non-error case, but instead get retried: https://github.com/matrix-org/synapse/blob/master/synapse/storage/_base.py#L249-L280

I'd really expect the exception to bubble up all the way to the client IP code, so I wonder if we're just swallowing it there?